### ISSUE TITLE
[IQ][Screen reader]Expanded State Not Announced for RTE Menus.

### ISSUE THREAD

**Issue by anilkolliboina2026-netizen** [tinymce/tinymce#10996]
## 📝 Provide detailed reproduction steps (if any)
Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](https://fiddle.tiny.cloud) or similar.

Repro Steps:
Open URL: https://stage.infosec-iq.com/Sites/index
Login with valid credentials.
Navigate to new template button and acivate with enter key template editer screen will appear.
Navigate to the Rich Text Editor (RTE).

Use the Tab key to move focus to a toolbar menu button (e.g., Font Size or Font Style).
Activate the menu using Enter or Space.
Observe the screen reader announcement.

### ✔️ Expected result
When the menu is expanded, the screen reader should announce “expanded” . The component should properly expose its state using appropriate ARIA attributes such as aria-expanded="true" when expanded and aria-expanded="false" when collapsed.

### ❌ Actual result
When the menu is collapsed, the screen reader announces the collapsed state.
When the menu is expanded, no announcement is made indicating that the menu is expanded.

## 📃 Other details
- Browser: …
- OS: …
- First affected version: …
- Worked in version: …

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by lostkeys** [comment]
Hi @anilkolliboina2026-netizen. We cannot log into someone else's app and debug it. We need a replication case that does not require any login and that is isolated to TinyMCE, for example a [TinyMCE fiddle](https://fiddle.tiny.cloud).
We also need the TinyMCE version used.
